website/docs/r/network_interface_attachment.html.markdown

```diff
@@ -59,31 +59,31 @@ data "alicloud_images" "default" {
 }
 
 resource "alicloud_instance" "instance" {
-  count = "${var.number}"
+  count             = "${var.number}"
   availability_zone = "${data.alicloud_zones.default.zones.0.id}"
-  security_groups = ["${alicloud_security_group.group.id}"]
+  security_groups   = ["${alicloud_security_group.group.id}"]
 
-  instance_type = "${data.alicloud_instance_types.instance_type.instance_types.0.id}"
-  system_disk_category = "cloud_efficiency"
-  image_id             = "${data.alicloud_images.default.images.0.id}"
-  instance_name        = "${var.name}"
-  vswitch_id = "${alicloud_vswitch.vswitch.id}"
+  instance_type              = "${data.alicloud_instance_types.instance_type.instance_types.0.id}"
+  system_disk_category       = "cloud_efficiency"
+  image_id                   = "${data.alicloud_images.default.images.0.id}"
+  instance_name              = "${var.name}"
+  vswitch_id                 = "${alicloud_vswitch.vswitch.id}"
   internet_max_bandwidth_out = 10
 }
 
 resource "alicloud_network_interface" "interface" {
-  count = "${var.number}"
-  name = "${var.name}"
-  vswitch_id = "${alicloud_vswitch.vswitch.id}"
-  security_groups = [ "${alicloud_security_group.group.id}" ]
+  count           = "${var.number}"
+  name            = "${var.name}"
+  vswitch_id      = "${alicloud_vswitch.vswitch.id}"
+  security_groups = ["${alicloud_security_group.group.id}"]
 }
 
 resource "alicloud_network_interface_attachment" "attachment" {
-  count = "${var.number}"
-  instance_id = "${element(alicloud_instance.instance.*.id, count.index)}"
+  count                = "${var.number}"
+  instance_id          = "${element(alicloud_instance.instance.*.id, count.index)}"
   network_interface_id = "${element(alicloud_network_interface.interface.*.id, count.index)}"
 }
-...
+# ...
 ```
```

Comment:
I think the line 20 and the line 86 are redundant and them should be removed.

Reply:
OK, it's done. Thanks for reviewing.